bin/hardening/3.5.4.1.1_net_fw_default_policy_drop.sh

```diff
@@ -28,7 +28,7 @@ audit() {
     if [ "$FNRET" != 0 ]; then
         crit "$PACKAGE is not installed!"
     else
-        ipt=$($SUDO_CMD "$FW_CMD" -nL 2>/dev/null || true)
+        ipt=$($SUDO_CMD "$FW_CMD" -w 10 -nL 2>/dev/null || true)
```

Comment:
Can you please set the timeout in a variable, so it can be easily overwritten in config

Reply:
PR amended

Reply:
:+1: